tests/test_clip.py

```diff
@@ -181,6 +181,70 @@ def test_ref_default(self):
             otio.schema.ExternalReference()
         )
 
+    def test_multi_ref(self):
+        cl = otio.schema.Clip()
+
+        self.assertEqual(
+            otio.schema.ClipMediaRepresentation.DEFAULT_MEDIA,
+            cl.active_media_reference
+        )
+        self.assertIsOTIOEquivalentTo(
+            cl.media_reference,
+            otio.schema.MissingReference()
+        )
+
+        mrs = cl.media_references
+        self.assertIsOTIOEquivalentTo(
+            mrs[otio.schema.ClipMediaRepresentation.DEFAULT_MEDIA],
+            otio.schema.MissingReference()
+        )
+
+        cl.media_references = {
+            otio.schema.ClipMediaRepresentation.DEFAULT_MEDIA:
+                otio.schema.ExternalReference(),
+            otio.schema.ClipMediaRepresentation.DISK_HIGH_QUALITY_MEDIA:
+                otio.schema.GeneratorReference(),
+            otio.schema.ClipMediaRepresentation.DISK_PROXY_QUALITY_MEDIA:
+                otio.schema.ImageSequenceReference(),
+            otio.schema.ClipMediaRepresentation.CLOUD_HIGH_QUALITY_MEDIA:
+                otio.schema.MissingReference(),
+            otio.schema.ClipMediaRepresentation.CLOUD_PROXY_QUALITY_MEDIA:
+                otio.schema.ExternalReference()
+        }
+
+        mrs = cl.media_references
+        self.assertIsOTIOEquivalentTo(
+            mrs[otio.schema.ClipMediaRepresentation.DEFAULT_MEDIA],
+            otio.schema.ExternalReference()
+        )
+        self.assertIsOTIOEquivalentTo(
+            mrs[otio.schema.ClipMediaRepresentation.DISK_HIGH_QUALITY_MEDIA],
+            otio.schema.GeneratorReference()
+        )
+        self.assertIsOTIOEquivalentTo(
+            mrs[otio.schema.ClipMediaRepresentation.DISK_PROXY_QUALITY_MEDIA],
+            otio.schema.ImageSequenceReference()
+        )
+        self.assertIsOTIOEquivalentTo(
+            mrs[otio.schema.ClipMediaRepresentation.CLOUD_HIGH_QUALITY_MEDIA],
+            otio.schema.MissingReference()
+        )
+        self.assertIsOTIOEquivalentTo(
+            mrs[otio.schema.ClipMediaRepresentation.CLOUD_PROXY_QUALITY_MEDIA],
+            otio.schema.ExternalReference()
+        )
+
+        cl.active_media_reference = \
+            otio.schema.ClipMediaRepresentation.DISK_HIGH_QUALITY_MEDIA
+        self.assertIsOTIOEquivalentTo(
+            cl.media_reference,
+            otio.schema.GeneratorReference()
+        )
+        self.assertEqual(
+            otio.schema.ClipMediaRepresentation.DISK_HIGH_QUALITY_MEDIA,
+            cl.active_media_reference
+        )
```

Comment:
We may just want to add some test cases around behavior when `active_media_reference` has no appropriate equivalent in the `media_references` dictionary.